.travis.yml

```diff
@@ -35,16 +35,13 @@ jobs:
   include:
     # Run tests with the dependencies from composer.lock
     - php: 7.2
-    - php: 7.3
```

Comment:
so no tests on PHP 7.3 anymore ?

Reply:
No ... I thought that the lowest supported version (7.2) and the highest supported version (7.4) was enough. Do you agree?